packages/docs/docs/circuits-api/circuits.md

```diff
@@ -158,7 +159,7 @@ Outputs:
 - `control[2]`
 
 :::info
-Control fields are use to encode many small values into a single field element. This reduces the number of public signals needed to operate a circuit.
+Control fields are used to encode many small values into a single field element. This reduces the number of public signals needed to operate a circuit.
 :::
 
 ## User State Transition Proof
```

Comment:
do you mean to leave the 'TODO' on line 171?

Reply:
commented out the TODO in the document